src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@
      * "Can't get min value from list: method_input_list"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::valueOf)
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list"));
```

Comment:
```suggestion
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: " + numbers));
```
this way we can make message more informative